src/commands/infra/scaffold.ts

```diff
@@ -39,7 +41,19 @@ export const scaffoldCommandDecorator = (command: commander.Command): void => {
         await copyTfTemplate(opts.template, opts.name);
         await validateVariablesTf(path.join(opts.template, "variables.tf"));
         await scaffold(opts.name, opts.source, opts.version, opts.template);
-        await renameTfvars(opts.name);
+        // await renameTfvars(opts.name);
+
+        // Remove template files after parsing
+        fs.readdir(opts.name, (err, files) => {
```

Comment:
Consider moving this logic out to its own function.  This would enable you to write a unit test for it and also declutter this central function.